frameit/README.md

```diff
@@ -277,7 +277,7 @@ brew install imagemagick
 Please submit an issue on GitHub and provide information about your setup
 
 # Code of Conduct
-Help us keep `frameit` open and inclusive. Please read and follow our [Code of Conduct](https://github.com/fastlane/code-of-conduct).
+Help us keep `frameit` open and inclusive. Please read and follow our [Code of Conduct](https://github.com/fastlane/fastlane/CODE_OF_CONDUCT.md).
```

Comment:
Argh, we should have tested those links, it should have been https://github.com/fastlane/fastlane/blob/master/CODE_OF_CONDUCT.md